rstudio/rstudio
# make links displayed in terminal clickable

### Intent

- Fixes #6621
- RStudio 1.1 and 1.2 supported clickable links in the terminal, but this was lost in RStudio 1.3 (xterm.js moved it out of core library into a separate add-on)
- Also added a user preference for enabling/disabling clickable links in terminal

### Approach

- Load the xterm.js add-on that provides this functionality

### QA Notes

Generate URL-formatted output in terminal, e.g. `echo "https://www.rstudio.com". Then hover mouse over it and click, should load in default browser.

Turn off the "Clickable web links" option in Global Options / Terminal, create a new terminal, and try again. Should not be a clickable link.

Note that like most terminal options changing this one does not modify existing terminal instances, you must either reload the UI or create a new terminal to see the impact.
